en/style-guide.adoc

```diff
@@ -147,7 +147,7 @@ Incorrect
  unless tail? list-of-books [author-property: select list-of-books selected-index]
 ----
 
-*Function names* should strive to be single-word _verbs_, in order to express an action, though two or three words names are often necessary. More than three words should be avoided as much as possible. Variable naming conventions also apply to function names. A noun or an adjective followed by a question mark is also accepted. Often, it denotes that the return value is of `logic!` type, but this is not a strict rule, as it is handy to form single-word action names for retrieving a property (e.g. `length?`, `index?`). When forming function names with two or more words, always put the verb in the first position. If names are picked carefully for variables and function names, the code becomes almost self-documented, often reducing the need for comments.
+*Function names* should strive to be single-word _verbs_, in order to express an action, though two or three words names are often necessary. More than three words should be avoided as much as possible. Variable naming conventions also apply to function names. A noun or an adjective followed by a question mark is also accepted. Often, it denotes that the return value is of `logic!` type, but this is not a strict rule, as it is handy to form single-word action names for retrieving a property (e.g. `length?`, `index?`). When forming function names with two or more words, always start with the verb. If names are picked carefully for variables and function names, the code becomes almost self-documented, reducing the need for comments.
```

Comment:
"When naming a function, strive to use single-word _verbs_, ....". "Variable naming conventions also apply to function names" part IMO makes no sense (functions don't have a name by themselves, words that refer to them do) and should be removed. And then "If your words are named carefully, the code becomes almost self-documented...".